static/app/stores/guideStore.tsx

```diff
@@ -122,19 +130,21 @@ const storeConfig: GuideStoreDefinition = {
     cleanupActiveRefluxSubscriptions(this.unsubscribeListeners);
     window.removeEventListener('load', this.onURLChange);
 
-    if (this.browserHistoryListener) {
```

Comment:
Preference code churn 🤷🏼‍♂️

Reply:
Do you think someone would change it back? I am pretty confident it was like this because someone didn't know about the null traversal operator for function calls.

Reply:
No, I made it that way :D I prefer if statements for function calls, I find maybeFunction?.() annoying to read because it looks 90% like a function call, but it's not. So when I read some code that I never read before, it's just easier to see separate branches (I dont mind it for safe access)